.github/workflows/ci.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ "main" ]    
   pull_request:
-    branches: [ "main" ] 
+    branches: [ "main", "feature/sprint11" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)